app/actions.py

```diff
@@ -62,6 +62,76 @@ def __init__(self, program):
         self.debugUpperChangedRow = -1
         self.parser = app.parser.Parser()
         self.fileFilter(u'')
+    
+    def getMatchingBracketAddress(self):
```

Comment:
What do you think of changing 'Address' to 'RowCol'? Address is okay if you prefer to keep 'Address', but it takes me and extra moment to realized that it doesn't mean a memory address (e.g. 0x0000000043238080) where 'RowCol' I don't have to take that moment.

Reply:
RowCol looks better to me too.